packages/@okta/vuepress-theme-prose/components/News.vue

```diff
@@ -77,15 +77,87 @@
           Blog
         </p>
         <h3 class="news__article-title dont-break-out">
-          Streamline Your Okta Configuration in Angular Apps
+          {{ blogPostTitles[4] || 'Streamline Your Okta Configuration in Angular Apps' }}
         </h3>
         <a
           class="news__article-link dont-break-out"
-          href="https://developer.okta.com/blog/2023/03/07/angular-forroot"
+          :href="blogPostLinks[4] || 'https://developer.okta.com/blog/2023/03/07/angular-forroot'"
         >
           Read the blog post
         </a>
       </article>
     </div>
   </section>
 </template>
+
+<script>
+const axios = require('axios');
+
+export default {
+  name: 'News',
+  data() {
+    return {
+      blogPostTitles: [],
```

Comment:
Instead of empty data we can store the current news as default values. i.e.
```
articles: [
  {
     title: 'How Authentication and Authorization Work for SPAs',
     link: 'https://developer.okta.com/blog/2023/04/04/spa-auth-tokens',
     isBlog: true
  },
  {
     title: 'Podcast: Phishing-Resistant Authenticators with Megha Rastogi',
     link: 'https://www.youtube.com/watch?v=PiY5HDp0ABI',
     isBlog: false
  }
  ...
]

```

when we get RSS successfully we will replace articles [0, .. 4]. At that point we can use `v-for` to generate the whole news template for the articles array.

Reply:
Maybe an even better approach would be to store JSON file with articles and have a script to update it from RSS. When writers want to get a newer RSS they would run `yarn update-rss` which will fetch XML and update JSON and then just create PR and push the change.
The advantage is we can patch Blog/Video property easily, we can configure a single item manually and we can test new RSS.